doc/usecases/polyculture/map_specific_favorite_groups.md

```diff
@@ -0,0 +1,29 @@
+# Use Case: Map-Specific Favorite Groups
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user changes a set of favorite plant groups per map.
+- **Status:** Assigned
+- **Assignee:** Benjamin
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged in to the app.
+  - The plants layer is selected.
+  - At least one plant is placed.
+- **Main success scenario:**
+  - The user adds or removes a plant group from her list of favorites.
```

Comment:
Can you please also add to this use case that single plant can be added to favorites?

```suggestion
  - The user adds a plant or a plant group to her list of favorites.
```